src/test/java/uk/gov/hmcts/divorce/common/notification/Applicant1AppliedForFinalOrderNotificationTest.java

```diff
@@ -125,15 +136,83 @@ public void verifyApplicant1TemplateVarsWhenFinalOrderNotEligible() {
             eq(TEST_USER_EMAIL),
             eq(SOLE_APPLIED_FOR_FINAL_ORDER),
             argThat(allOf(
-                hasEntry(WILL_BE_CHECKED_WITHIN_2_DAYS, CommonContent.NO),
-                hasEntry(WILL_BE_CHECKED_WITHIN_14_DAYS, CommonContent.YES),
+                hasEntry(WILL_BE_CHECKED_WITHIN_2_DAYS, NO),
+                hasEntry(WILL_BE_CHECKED_WITHIN_14_DAYS, YES),
                 hasEntry(NOW_PLUS_14_DAYS, getExpectedLocalDate().plusDays(14).format(DATE_TIME_FORMATTER))
             )),
             eq(ENGLISH)
         );
         verify(commonContent).mainTemplateVars(data, 1L, data.getApplicant1(), data.getApplicant2());
     }
 
+    @Test
+    void shouldSendApplicant2SolicitorNotificationIfJointApplicationAndIsRepresented() {
+        CaseData data = validApplicant2CaseData();
+        data.setApplicationType(JOINT_APPLICATION);
+        data.getApplicant1().setSolicitorRepresented(YesOrNo.YES);
+        data.getApplicant2().setSolicitorRepresented(YesOrNo.YES);
+        data.getApplication().setIssueDate(LOCAL_DATE);
+        data.setFinalOrder(FinalOrder.builder()
+                .dateFinalOrderNoLongerEligible(getExpectedLocalDate().plusDays(30))
+                .applicant1AppliedForFinalOrder(YesOrNo.YES)
+                .applicant2AppliedForFinalOrder(YesOrNo.YES)
+                .build()
+        );
+        data.getApplicant1().setSolicitor(Solicitor.builder()
+                .email(TEST_SOLICITOR_EMAIL).build());
+
+        data.getApplicant2().setSolicitor(Solicitor.builder()
+                .email(TEST_SOLICITOR_EMAIL).build());
+
+        when(commonContent.basicTemplateVars(data, 1L)).thenReturn(getBasicTemplateVars());
+
+        notification.sendToApplicant1Solicitor(data, 1L);
+        notification.sendToApplicant2Solicitor(data, 1L);
```

Comment:
I think we should test these methods independently in separate test methods instead of within a single test. 

`notification.sendToApplicant1Solicitor(data, 1L);`

`notification.sendToApplicant2Solicitor(data, 1L);`